owncloudApp/src/main/java/com/owncloud/android/features/FeatureList.java

```diff
@@ -40,6 +40,15 @@
     static final private boolean SHOW_ON_FIRST_RUN = true;
     static final private boolean SHOW_ON_UPGRADE = false;
 
+    static final private int VERSION_CODE_BETA_2_15_1 = 21400201;
+    static final private FeatureItem INCONSISTENT_VERSION = new FeatureItem(
+            R.drawable.whats_new_warning,
+            R.string.welcome_feature_6_title,
+            R.string.welcome_feature_6_text,
+            "2.14.2",
```

Comment:
I know there are other hardcoded versions like "2.7.0" ones, it would be great to add some constants for those and these "2.14.2" and "0". Don't need to do the "2.7.0" changes here, up to you.